internal/cypress/sauce/runner.go

```diff
@@ -78,31 +84,53 @@ func (r *Runner) runSuites(fileID string) int {
 
 	// Collect results.
 	errCount := 0
-	for i := 0; i < len(r.Project.Suites); i++ {
-		r := <-results
-		if r.err != nil {
-			log.Err(r.err).Str("suite", r.suiteName).Msg("Suite failed.")
+	completed := 0
+	total := len(r.Project.Suites)
+	inprogress := total
+	exitCode := 0
```

Comment:
@valentyn88 what about using `passed`? and the `runSuites` method returns true/false regarding if all suites passed. We leave the exitCode  logic to the caller

Reply:
Good idea to keep logic regarding the exit code in one place.